criteo_tft/README.md

```diff
@@ -172,6 +172,14 @@ at least 2 hours to train, and the deep model more than 8 hours. You can use
 [Tensorboard](https://www.tensorflow.org/how_tos/summaries_and_tensorboard/) to
 follow the job's progress.
 
+#### How to run Tensorboard
+
+You can run Tensorboard using the command
```

Comment:
It's good to mention that tensorboard is installed with tensorflow.

(For example, if the user installed tensorflow in a virtual environment, they should run tensorboard inside that environment.)

Reply:
done